tasks/managed/embargo-check/embargo-check.yaml

```diff
@@ -99,14 +99,31 @@ spec:
             fi
             set -x
 
-            # Perform additional checks only for Vulnerability issues for issues.redhat.com
+            # Perform additional checks only for issues for issues.redhat.com
             if [ "$server" != "issues.redhat.com" ] ; then
                 continue
             fi
+
+            # public should be true if and only if the security field doesn't exist and the issue is accessible
+            # without authentication
+            public=false
+
+            if ! "$(jq '.fields | has("security")' <<< "$OUTPUT")" ; then
+                if curl -o /dev/null --retry 3 --fail "${API_URL}" ; then
+                    public=true
+                fi
+            fi
+
+            # Inject public key to each issues.redhat.com issue
```

Comment:
I find the commend confusing. It sounds like it would add it to multiple issues, but it's only adding it to the current issue. And it only happens if it's a issues.redhat.com issue because otherwise this whole block would be skipped on line 103, right? So I'd just say "Inject public key"?